tests/test_tiatoolbox.py

```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env python
 
-"""Tests for `tiatoolbox` package."""
+"""pytests for `tiatoolbox` package."""
```

Comment:
Very minor point. I thought 'Tests' was fine before. Starting the sentence with pytests sounds weird to me. :shrug:

Reply:
All we need is to be consistent, whether Tests or pytests

Reply:
This is done for most strings in this file. Shall we change all to Test?

Reply:
I will change all to `Tests` now

Reply:
Fixed [here](https://github.com/TIA-Lab/tiatoolbox/pull/35/commits/0f1081053337cdbaabe8205a779109c9e04d3f81).